tasks/broad/BamProcessing.wdl

```diff
@@ -273,10 +273,13 @@ task GatherSortedBamFiles {
     Float total_input_size
     Int compression_level
     Int preemptible_tries
+    Int additional_disk = 20
+    Int memory_multiplier = 1
   }
 
   # Multiply the input bam size by two to account for the input and output
-  Int disk_size = ceil(2 * total_input_size) + 20
+  Int disk_size = ceil(2 * total_input_size) + additional_disk
+  Int machine_mem_mb = ceil(3000 * memory_multiplier)
```

Comment:
We are gonna want to also scale the  -Xms and -Xmx params in the command block a few lines below. Usually, we set xmx about 500 less than machine_mem_mb and xms is usually 1000 less